multimodal/src/autogluon/multimodal/optimization/utils.py

```diff
@@ -99,7 +101,25 @@ def get_loss_func(
         if mixup_active:
             loss_func = SoftTargetCrossEntropy()
         else:
-            loss_func = nn.CrossEntropyLoss()
+            if loss_func_name is not None and loss_func_name.lower() == "focal_loss":
+                if OmegaConf.select(config, "focal_loss.alpha"):
+                    alpha = torch.tensor(OmegaConf.select(config, "focal_loss.alpha"))
+                else:
+                    alpha = None
+
+                if OmegaConf.select(config, "focal_loss.gamma"):
+                    gamma = OmegaConf.select(config, "focal_loss.gamma")
+                else:
+                    gamma = 2.0
+
+                if OmegaConf.select(config, "focal_loss.reduction"):
+                    reduction = OmegaConf.select(config, "focal_loss.reduction")
+                else:
+                    reduction = "mean"
+
+                loss_func = FocalLoss(alpha=alpha, gamma=gamma, reduction=reduction)
```

Comment:
How about changing to this? I think it's more concise.

```
loss_func = FocalLoss(
    alpha=OmegaConf.select(config, "focal_loss.alpha"),
    gamma=OmegaConf.select(config, "focal_loss.gamma", default=2.0),
    reduction=OmegaConf.select(config, "focal_loss.reduction", default="mean"),
)
```

Reply:
Oh that's better

Reply:
the issue is that for alpha, if it is not none, then we have to cast it with `torch.tensor(...)`. Otherwise, we don't cast it and assign `alpha = None`. I can fit this in one line but it would be a little long.

Reply:
like this: 

```
loss_func = FocalLoss(
    alpha=torch.tensor(OmegaConf.select(config, "focal_loss.alpha")) if OmegaConf.select(config, "focal_loss.alpha") is not None else None,
...
)
```
```

Reply:
Can I abbreviate the `OmegaConf.select` here to like `config.focal_loss.alpha`?

Reply:
We need to use `select` here since this is for backward compatibility, i.e., you have a new key added.

Reply:
I think it's better to put `torch.tensor(alpha)` into the forward function of `FocalLoss`.

Reply:
ok that makes sense.